pytest.ini

```diff
@@ -21,13 +21,29 @@ testpaths =
     # API
     forge/test/test_api.py
 
+    # Model Tests (CNN)
+    forge/test/model_demos/high_prio/cnn/pytorch
+
+    # Model Tests (NLP)
+    forge/test/model_demos/high_prio/nlp/pytorch
+
     # MNIST Linear
     forge/test/mlir/mnist/test_inference.py
     forge/test/mlir/mnist/training/test_training.py
     forge/test/mlir/test_training.py
 
     # Llama
     forge/test/mlir/llama/test_llama_inference.py::test_llama_inference
+    forge/test/mlir/llama/tests/test_llama_embedding.py
```

Comment:
Can we add: `forge/test/mlir/llama/tests`, instead of specifying each tested separately?

Reply:
Updated.